### PR TITLE
Reactdev

### PR DESCRIPTION
**Pull request by krisdo**
--circular avatar
--header
--condition superhost badge
--joined month and year

### CODE REVIEW

File: client/components/Header.jsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import { Image } from 'semantic-ui-react';
+
+const badge = `${window.location.href}superhost_badge.png`;
+
+const badgePosition = {
+  position: 'absolute',
+  left: '47px',
+  top: '35px'
+};
+
```
Comment (samasastudio) [inline client/components/Header.jsx:11]: 👍👍👍

File: database-mongodb/data.js [hunk 2]
```diff
@@ -30,7 +30,7 @@ const generateHosts = () => {
       "listingId": [shuffledIds.pop()],
       "fullName": faker.name.findName(),
       "joined": faker.date.past(),
-      "photoUrl": faker.image.people(),
+      "photoUrl": faker.image.avatar(),
       "superhost": faker.random.boolean(),
```
Comment (samasastudio) [inline database-mongodb/data.js:34]: what brought about this change?

File: client/components/Header.jsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+const Header = ({name, superhost, img, joined}) => (
+  <div style={{ display: 'flex', marginBottom: '24px'}}>
+  <Image style={{marginRight: '16px'}} circular width={56} src={img}/>
+  {superhost}
+  {superhost ? <Image height={23} src={badge} style={badgePosition} /> : null}
+  {name !== null ? <div><h2 style={{marginBottom: '0px'}}>Hosted by {name.split(' ')[0]}</h2><p style={{color: 'rgb(113, 113, 113)'}}>Joined in {joined.split(' ')[1]} {joined.split(' ')[3]} </p> </div>: null }
+  </div>
```
Comment (samasastudio) [inline client/components/Header.jsx:18]: will you be doing a mix of Semantic UI & Styled Components going forward or just Semantic?

File: database-mongodb/data.js (hunk 2 above)
Comment (krisdo) [reply database-mongodb/data.js:34]: I realized the original photos were groups of people. I needed solo photos hence the name avatar.

File: client/components/Header.jsx (hunk 3 above)
Comment (krisdo) [reply client/components/Header.jsx:18]: I wanted to incorporate styled components but I had a lot of issues with the installation for proxy and that wasted a good chunk of my time lol. I will think about it later. I am having so much fun learning semantics tho! Their base font is similar to airbnb's

**Pull request merged**